app/ts/background/windows/confirmTransaction.ts

```diff
@@ -39,19 +39,36 @@ export const isConfirmTransactionFocused = async () => {
 	return popup.tab.active
 }
 
+const getPendingTransactionByidentifier = async (uniqueRequestIdentifier: UniqueRequestIdentifier) => {
+	return (await getPendingTransactions()).find((tx) => doesUniqueRequestIdentifiersMatch(tx.uniqueRequestIdentifier, uniqueRequestIdentifier))
+}
+
 export async function resolvePendingTransaction(simulator: Simulator, websiteTabConnections: WebsiteTabConnections, confirmation: TransactionConfirmation) {
-	const pendingTransaction = await removePendingTransaction(confirmation.data.uniqueRequestIdentifier)
+	console.log(confirmation)
```

Comment:
this may be a debugging helper that can be removed

Reply:
yeah, its removed in later pr's